src/modules/loadFullPage.js

```diff
@@ -0,0 +1,44 @@
+import fullHeader from './header';
+import fullFooter from './footer';
+import contacPage from './contact';
+import menuPage from './menu';
+import mainSection from './main';
+import staffPage from './staff';
+
+const pageInit = () => {
+  const main = document.getElementById('content');
+  const header = fullHeader();
+  const mainSect = mainSection();
+  const footer = fullFooter();
+
+  main.appendChild(header);
+  const home = document.getElementById('About');
+  home.addEventListener('click', () => {
+    mainSect.innerHTML = '';
+    mainSect.appendChild(mainSection());
+  });
+
+  const menu = document.getElementById('Menu');
+  menu.addEventListener('click', () => {
+    mainSect.innerHTML = '';
+    mainSect.appendChild(menuPage());
+  });
+
+  const staff = document.getElementById('Staff');
+  staff.addEventListener('click', () => {
+    mainSect.innerHTML = '';
+    mainSect.appendChild(staffPage());
+  });
+
+  const contact = document.getElementById('Contact');
+  contact.addEventListener('click', () => {
+    mainSect.innerHTML = '';
+    mainSect.appendChild(contacPage());
+  });
+
+  main.appendChild(mainSect);
+  main.appendChild(footer);
+  return main;
+};
+
```

Comment:
i guess the problem explained above has to do with the way you organised your menus click here. Each click should be independent, not wrapped in one giant module. You can there leave just `pageInit()` function here, and relocate the others (they should be independent because each generates it's own page) to `app.js`

This is just a suggestion. But not matter what you do, just make it work😊

Reply:
Thanks a lot. It is working now. I just made a mistake in the styling. I also solved the problem with the giant module. I made a new function inside app.js